mb_modelbase/server/modelcache.py

```diff
@@ -0,0 +1,93 @@
+import abc
+from pymemcache.client import base as memcBase
+import redis
+
+class Cache():
+    def __init__(self, *args, **kwargs):
```

Comment:
Please document where appropriate, also the subclasses below.

Reply:
See above.